templates/test_templates.sh

```
@@ -0,0 +1,6 @@
+#!/bin/bash
+# templates.sh - Armbian Config V3 module
+
+if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
+	echo "place Holder
```

Shellcheck (0.10.0)

[error] Couldn't parse this double quoted string. Fix to allow more checks.

(SC1073)